Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM hathitrust/feed_base:buster
+
+RUN apt-get install -y libtest-class-perl libswitch-perl libtest-spec-perl libtest-mockobject-perl
+
+RUN git config --global url."https://github.com/".insteadOf git@github.com:
```

Comment:
This probably doesn't hurt, but is it necessary with the instructions now indicating you should get the submodules outside the scope of building the image?

Reply:
No, I'll remove them.